### ISSUE TITLE
Brand request: Bluesky

### ISSUE THREAD

**Issue by cbracco** [FortAwesome/Font-Awesome#19810]
### Description

The social network is still in private beta, but it probably has a logo and will likely go public at some point.

### Official website

https://bsky.app/

### Icon request checklist

- [X] The title starts with "Brand request: ", is followed by the requested brand name
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/brands#faqs)

**Comment by santumerino** [comment]
I'm not sure what the etiquette on "bumping" issues is, but I'd like to +1 this request.

**Comment by akrabu** [comment]
@santumerino Just thumbs up the OP!  That's how "voting" works here.  :)

**Comment by sensibleworld** [comment]
Does Bluesky have an actual logo?

**Comment by akrabu** [comment]
Basically, a blue sky:

![unnamed](https://github.com/FortAwesome/Font-Awesome/assets/6868732/25c36472-2da4-4529-8fe0-4ba245a5db59)

Which I'm uncertain how that might convert to a Font Awesome icon.  Not certain if it's really possible.  Maybe like a black square with little clouds in it?  I don't know how it would work, honestly.

For their favicon, they use this:

![Unknown](https://github.com/FortAwesome/Font-Awesome/assets/6868732/b6b127f5-c6a5-4e0b-b411-0d92f51afff9)

So I've personally just been using [this icon](https://fontawesome.com/icons/square?f=classic&s=solid) to represent it.

**Comment by cbracco** [comment]
This is what we’ve been using in the meantime.

![bluesky-icon](https://github.com/FortAwesome/Font-Awesome/assets/1480677/c88d074a-50c0-4fd7-a430-74109fbbcbf1)

**Comment by cbracco** [comment]
We found [this logo icon/image](https://blueskyweb.xyz/images/logo-64x64.jpg) a few months back, and re-created the birds in this icon/image as an SVG and are using that for now.

**Comment by tagliala** [comment]
Hello, it would important for this request to have a single color, official version of the logo

**Comment by jasongaylord** [comment]
An official guideline for an icon may come from emailing support here: https://blueskyweb.zendesk.com/hc/en-us/articles/19002493054861-Miscellaneous

**Comment by kpossibles** [comment]
I emailed Bluesky support so they can think about creating an icon soon...

**Comment by chkuendig** [comment]
https://bsky.app/ has the following `og:image` property: `https://bsky.app/static/social-card-default.png`

This is probably the closest we get to a proper icon currently. It also matches the iconography in the hero image at https://bsky.app/profile/bsky.app and looks simple enough to vectorize or turn grayscale.



![https://bsky.app/static/social-card-default.png](https://bsky.app/static/social-card-default.png)

**Comment by imgwrx** [comment]
bluesky is a client build on the at protocol. with many 3rd party clients like graysky and deck.blue stealing the show. so maybe just use the at protocol logo?

https://atproto.com/

**Comment by chkuendig** [comment]
Funnily enough that page just uses `fa-square` as the logo for Bluesky: 
<img width="336" alt="image" src="https://github.com/FortAwesome/Font-Awesome/assets/748698/906c8c03-5e6e-4576-b9da-1a017e487be6">

**Comment by pfrazee** [comment]
This has not been officially announced but it will be soon. (Users watch our github repo too closely to keep a secret!)

![logo-black](https://github.com/FortAwesome/Font-Awesome/assets/1270099/a8ec9eb5-e2ee-495e-af93-d178351841bb)

**Comment by fluentmoheshwar** [comment]
> This has not been officially announced but it will be soon. (Users watch our github repo too closely to keep a secret!)
> 
> ![logo-black](https://private-user-images.githubusercontent.com/1270099/290378213-a8ec9eb5-e2ee-495e-af93-d178351841bb.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMzMTczOTYsIm5iZiI6MTcwMzMxNzA5NiwicGF0aCI6Ii8xMjcwMDk5LzI5MDM3ODIxMy1hOGVjOWViNS1lMmVlLTQ5NWUtYWY5My1kMTc4MzUxODQxYmIuc3ZnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMjNUMDczODE2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWQzNmJhNjkzMDFjOWRhMGZlOTZlM2RlNWI0MmEzMDYxOTQyOTRjMzQwMzQxYzE5NTE2MDQ3MDgyMjAwZTUzOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.aR_9V4BKgRsLgev-CzFSnOyHGWB35druaaepZMTnDLU)

They officially announced this please add this ASAP!

**Comment by haideralipunjabi** [comment]
Waiting for this logo!!

**Comment by Dominic-t3ch** [comment]
I use font awesome for the social media buttons on my blog. I hoped an official Bluesky logo to add it to my website. Now that is official, I hope to see it soon in the Font awesome icons set.

Thanks

**Comment by fishcharlie** [comment]
This issue went from 39th on the leaderboard yesterday to currently 6th: https://fontawesome.com/leaderboard/brands

Although I understand that a lot of factors go into choosing which icons the FontAwesome team spends time integrating. I hope they are at least taking note of the momentum & popularity of this request.

I hope to see this included in the FontAwesome collection really soon. If there is anything I can do to support this request or help move it forward please let me know.

**Comment by sensibleworld** [comment]
Hey folks. This icon has been added to our build system and will be released with the next version of Font Awesome. (Unfortunately, I don't yet have an estimate on when that will be.)

**Comment by pfrazee** [comment]
Thank you! I feel honored to work on a project that'll be included in font awesome

**Comment by YasoKuuhl** [comment]
I use Font Awesome for my website, and as a Bluesky user, I'm looking forward so much to being able to add a Bluesky icon to my social media buttons soon!

**Comment by simontomes** [comment]
This is an excellent idea. The Bluesky logo available via Font Awesome feels like a very natural progression for the Bluesky ecosystem and community.

**Comment by yovantichrist** [comment]
> Funnily enough that page just uses `fa-square` as the logo for Bluesky: <img alt="image" width="336" src="https://private-user-images.githubusercontent.com/748698/273194592-906c8c03-5e6e-4576-b9da-1a017e487be6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ3OTM1NDQsIm5iZiI6MTcwNDc5MzI0NCwicGF0aCI6Ii83NDg2OTgvMjczMTk0NTkyLTkwNmM4YzAzLTVlNmUtNDU3Ni1iOWRhLTFhMDE3ZTQ4N2JlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwOVQwOTQwNDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDY5YzNjMGFhNGViZGZiNjQ4YzNiY2YyNmI4Yzk1MzQ3OGU0ZDM3N2MxYzQ2MDJjMWU2N2NjOWI0ZDU2YTA4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7bKP41NwK9QOmdsxGNIvoEi3wnBt8z_krCQ2V2HVW8s">

That's the icon I've been using for bluesky in my website for the last few months too xD

**Comment by Dominic-t3ch** [comment]
Hey guys, do you know IcoMoon ? I use this on my blog. It let you select fonts from some sources and to import as well icons . I imported the SVG provided by @pfrazee in my IcoMoon project than generated my new fonts. My blog show now a Bluesky button !

[https://icomoon.io/](url)

![2024-01-10 - Bouton Bluesky Dominic tech v2](https://github.com/FortAwesome/Font-Awesome/assets/26441607/fe43c2fe-00b7-4643-9263-ade6a7267fa8)

**Comment by george-gca** [comment]
If you need a few icons, icomoon is a good solution, but if you want a whole range of icons and symbols it becomes tedious to add all of them. In this way FontAwesome fits my solution better. But I use it with [PurgeCSS](https://purgecss.com/) to drop all the icons that I don't use.

**Comment by owenblacker** [comment]
Linking #15399

**Comment by tyteen4a03** [comment]
Any ETA on this?

**Comment by WentTheFox** [comment]
FYI, Bluesky just opened up registration without invites, an influx of users is expected

https://bsky.social/about/blog/02-06-2024-join-bluesky

**Comment by afbase** [comment]
just a heads up BlueSky now has a media press kit with logos.  

1. Logo Google Drive: https://drive.google.com/drive/folders/1RDpuQOQMfM9mXQ61wUYWNZUbgvDc8r-n

The SVG is:
```html
<svg width="1024" height="1024" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M351.121 315.106C416.241 363.994 486.281 463.123 512 516.315C537.719 463.123 607.759 363.994 672.879 315.106C719.866 279.83 796 252.536 796 339.388C796 356.734 786.055 485.101 780.222 505.943C759.947 578.396 686.067 596.876 620.347 585.691C735.222 605.242 764.444 670.002 701.333 734.762C581.473 857.754 529.061 703.903 515.631 664.481C513.169 657.254 512.017 653.873 512 656.748C511.983 653.873 510.831 657.254 508.369 664.481C494.939 703.903 442.527 857.754 322.667 734.762C259.556 670.002 288.778 605.242 403.653 585.691C337.933 596.876 264.053 578.396 243.778 505.943C237.945 485.101 228 356.734 228 339.388C228 252.536 304.134 279.83 351.121 315.106Z" fill="#1185FE"/>
</svg>
```

**Comment by afbase** [comment]
> Hey folks. This icon has been added to our build system and will be released with the next version of Font Awesome. (Unfortunately, I don't yet have an estimate on when that will be.)

@sensibleworld Will it reflect the official logo as shared https://github.com/FortAwesome/Font-Awesome/issues/19810#issuecomment-1935251473 ?

**Comment by colons** [comment]
hi. it has been almost three months since a bluesky dev dropped their official icon in this thread, and fontawesome users still don't have access to this icon. i feel a bit silly running a public-facing website that just uses a cloud when an official logo has been around this long. please try to get a release out with this included soon

**Comment by justinvforvendetta** [comment]
^ this. this pr was opened almost a year ago, and there are several svg's available of the official logo all over the internet. what is the delay?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for your interest in this PR, **labelled as "in progress"**.

You may be interested in taking a look at our [Requesting Icons FAQs](https://fontawesome.com/support#requesting-icons), they contain a lot of information about how requests work

**Comment by YasoKuuhl** [comment]
I’m also very interested in knowing when the Bluesky icon will finally be released with the next version of Font Awesome. Is there a rough timeline/date when this will be the case?

**Comment by sensibleworld** [comment]
Hey folks. I want to apologize for how long it's taking to get the Bluesky logo out. It's ready. It's in our next release. We've just been flat out on other projects and haven't had time to run the release process. Really, really, hoping to get it out next week.

**Comment by rothgar** [comment]
Thank you for all the work you do! TIL about the icon leader board. It's really cool you show requests and what's in process :pray:  https://fontawesome.com/leaderboard/in-progress

**Comment by YasoKuuhl** [comment]
Any news on when the next version of Font Awesome will be released? Because “next week” is going to be two weeks ago, and it would be great if we could add the Bluesky icon to our websites soon. It would certainly be a nice Easter surprise🙂

**Comment by sensibleworld** [comment]
I know, I know! We hit an unexpected snag in another project, which delayed the work in getting the latest FA release out. Sounds like another excuse, but we're a small team, and we can only focus on so many things at once 😅. We're all frustrated with the delay.

**Comment by Matevoun** [comment]
Nothing about Bluesky or Threads logos again for fontawesome ?

**Comment by sensibleworld** [comment]
@Matevoun The Bluesky logo is coming! The Threads logo already exists in Font Awesome.

**Comment by Matevoun** [comment]
@sensibleworld Niccce ;)

For Threads i didn't found it ... :(

**Comment by WentTheFox** [comment]
Can we please get this thread locked to collaborators, I only want to be notified when this releases not when another person asks "is this released yet"

**Comment by george-gca** [comment]
This was finally released! Just saw the [6.5.2](https://github.com/FortAwesome/Font-Awesome/releases/tag/6.5.2) version [release notes](https://fontawesome.com/changelog).

**Comment by fluentmoheshwar** [comment]
close the issue then! :)

**Comment by sensibleworld** [comment]
Sorry about the delay folks! https://fontawesome.com/icons/bluesky?f=brands&s=solid

**Comment by yovantichrist** [comment]
It's just me or it feels bigger than the rest of icons?

**Comment by Br3nnabee** [comment]
> It's just me or it feels bigger than the rest of icons?

Yeah same experience here, very irritating.

**Comment by somethingSTRANGE** [comment]
When used beside the other social icons, the Bluesky icon does look large. When I scale it down to about 90% its original size, it looks closer to the other icons in size and weight.

**Comment by skywalker023** [comment]
Can you please fix the size? And please add the square icon for bluesky as well 🙏🏻

**Comment by candalfigomoro** [comment]
@sensibleworld Would it be possible to have a `square-bluesky` (square shaped) icon?

**Comment by sensibleworld** [comment]
We adjusted the size of the icon just a bit, and also released a square version in 6.7.1

https://fontawesome.com/icons/square-bluesky?f=brands&s=solid
https://fontawesome.com/icons/bluesky?f=brands&s=solid

**Comment by skywalker023** [comment]
Thanks so much for the quick update!
